.kitchen.yml

```diff
@@ -27,3 +27,108 @@ suites:
       inspec_tests:
         - test/smoke/default
     attributes:
+  - name: nexus_properties_configured
+    run_list:
+      - recipe[nexus_repository_manager::default]
+    verifier:
+      inspec_tests:
+        - test/smoke/nexus_properties_configured
+    attributes:
+      nexus_repository_manager:
+        nexus_properties:
+          application_port: 8082
+          application_host: 127.0.0.1
+          nexus_context_path: "/test-context"
+          nexus_loadAsOSS: true
+          nexus_clustered: true
```

Comment:
How is loadAsOSS and clustered true?

Reply:
Both have been changed to false for this test (cd8305d).